war/src/main/js/pages/cloud-set/index.js

```diff
@@ -0,0 +1,12 @@
+import { registerSortableTableDragDrop } from "@/sortable-drag-drop";
+
+document.addEventListener("DOMContentLoaded", function () {
+  document.querySelectorAll("tbody").forEach((table) =>
+    registerSortableTableDragDrop(table, function () {
+      YAHOO.util.Dom.removeClass(
```

Comment:
why is this using `YAHOO.util.Dom` rather than just 
```javascript
document.getElementById("saveButton").classList.remove("jenkins-hidden")
```

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
That's me being rusty at javascript, just looked up for similar usages in the jenkins code base ;)